include/seastar/util/process.hh

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is open source software, licensed to you under the terms
+ * of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.  You may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Copyright (C) 2022 Kefu Chai ( tchaikov@gmail.com )
+ */
+
+#pragma once
+
+#include <sys/types.h>
+#include <string_view>
+#include <variant>
+#include <vector>
+#include <seastar/core/iostream.hh>
+#include <seastar/core/sstring.hh>
+
+namespace seastar::experimental {
+
+class process {
+public:
+    process(pid_t pid, int stdout, int stderr);
+    input_stream<char>& stdout() {
+        return _stdout;
+    }
+    input_stream<char>& stderr() {
+        return _stderr;
+    }
```

Comment:
What about stdin?

I suppose it can be left to a future iteration, no compatibility problem.

Reply:
added the stdin support.